app/services/duty_options/additional_duty/excise.rb

```diff
@@ -0,0 +1,26 @@
+module DutyOptions
+  module AdditionalDuty
+    class Excise < DutyOptions::AdditionalDuty::Base
+      PRIORIY = 6
+
+      include CommodityHelper
+
+      def duty_calculation_row
+        presented_row = []
+        presented_row << I18n.t(
+          'duty_calculations.options.excise_duty_html',
+          additional_code_description: excise_description,
+        ).html_safe
```

Comment:
very minor but I don't think `.html_safe`  is needed - a `_html` key will get marked html_safe automatically

Reply:
Feeexed